.github/workflows/test-integration.yml

```
@@ -115,3 +100,14 @@ jobs:
         working-directory: ./crates/integration-tests
         run: |
           yarn test:browser
+          
+      - name: Alert on nightly build failure  
+        uses: JasonEtco/create-an-issue@v2
+        if: ${{ failure() && github.event_name == 'schedule' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          WORKFLOW_NAME: ${{ github.workflow }}
+          WORKFLOW_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        with:
+            update_existing: true
+            filename: .github/NIGHTLY_BUILD_FAILURE.md
```

```suggestion
            filename: .github/NIGHTLY_TEST_FAILURE.md
```

The name currently implies that it's related to the nightly release.